core/trino-main/src/main/java/io/trino/SystemSessionProperties.java

```diff
@@ -261,6 +262,11 @@ public SystemSessionProperties(
                         "Maximum execution time of a query",
                         queryManagerConfig.getQueryMaxExecutionTime(),
                         false),
+                durationProperty(
+                        QUERY_MAX_PLANNING_TIME,
+                        "Maximum planning time of a query",
+                        queryManagerConfig.getQueryMaxPlanningTime(),
```

Comment:
please add validation here too (`>=`).

Reply:
`Duration` already validates the negative numbers